### PR TITLE
feat: create the managed zone when using external-dns

### PR DESCRIPTION
**Pull request by cagiti**
Signed-off-by: Cai Cooper <caicooper82@gmail.com>

### CODE REVIEW

File: pkg/jx/cmd/opts/common.go [hunk 1]
```diff
@@ -1087,6 +1088,16 @@ func (o *CommonOptions) CertManagerClient() (certmngclient.Interface, error) {
 	return o.certManagerClient, nil
 }
 
+// GetNameServers returns the name servers for the clusters domain
+func (o *CommonOptions) GetNameServers() []string {
+	return o.nameServers
+}
+
```
Comment (warrenbailey) [inline pkg/jx/cmd/opts/common.go]: Doesn't seem any need for these get/set functions, could you just export the Struct variable instead?

Comment (cagiti) [reply pkg/jx/cmd/opts/common.go]: True, I was maintaining some function parity...will refactor 😀

**Comment by hferentschik** [comment]
this PR needs a rebase now that #4248 got merged.

**Comment by cagiti** [comment]
@houndci-bot retest

**Comment by cagiti** [comment]
@houndci-bot

**Comment by cagiti** [comment]
Houndci check unavailable, reverted to a manual merge

**Pull request merged**